website/events-listener/package.json

```diff
@@ -3,29 +3,30 @@
   "version": "0.1.2",
   "description": "",
   "main": "src/index.ts",
-  "//type": "module",
+  "type": "module",
   "scripts": {
     "start": "node --experimental-specifier-resolution=node dist/index.js",
-    "start:dev": "nodemon src/index.ts",
+    "start:dev": "ts-node-dev src/index.ts",
```

Comment:
Good thinking! I somehow fixated that we need precisely `nodemon` package and nothing but it for some reason 😊

Reply:
Yeah. There was a little trouble. When we don't use `type: module` in package.json the app breaks in `start` script. Otherwise the app breaks in `start:dev` script if we use nodemon